src/App.jsx

```diff
@@ -17,6 +17,7 @@ import ParticipantSubmissionPage from "./components/views/ParticipantSubmissionP
 import "react-toastify/dist/ReactToastify.css";
 import PageNotFound from "./components/views/PageNotFound";
 import UserProfileFormPage from "./components/views/UserProfileFormPage";
+import UserProfilePage from "./components/views/userProfilePage";
```

Comment:
Always start the naming of your component files in caps
"./components/views/userProfilePage"; 
should be
"./components/views/UserProfilePage";

Reply:
branch naming does not follow convention. Please prepend
`feat` or `feature` if it's a feature branch
`chore` if it's a chore branch
`bug` or `bugfix` if it's a bugfix branch

Reply:
We need to work on rendering and incrementing the page count on the frontend

Reply:
It was probably a mistake, will work on it.